recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "python-json-logger" %}
-{% set version = "2.0.1" %}
-{% set sha256 = "f26eea7898db40609563bed0a7ca11af12e2a79858632706d835a0f961b7d398" %}
+{% set version = "2.0.7" %}
+{% set sha256 = "23e7ec02d34237c5aa1e29a070193a4ea87583bb4e7f8fd06d3de8264c4b2e1c" %}
```

Comment:
it is used once only so maybe do not use an extra variable for this

Reply:
Using this pattern seems to be common in our code. It would be nice to see a standard for how we write these, as there is quite a bit of variation.